presto-docs/src/main/sphinx/installation/jmx.rst

```diff
@@ -0,0 +1,67 @@
+===================
+Monitoring with JMX
+===================
+
+Presto exposes a large number of different metrics via the Java Management
+Extensions (JMX). JConsole, supplied with the JDK, VisualVM, and many other
+tools can be used to access the metrics in a client application. Many monitoring
+solutions support JMX.
+
+You can configure the ports used by JMX in ``jvm.config`` as described in
```

Comment:
This is controlled in config properties, not JVM.
```
You can configure the ports used by JMX as described in :ref:`config_properties`.

Reply:
updated